### ISSUE TITLE
How to debug utilsScript?

### ISSUE THREAD

**Issue by tgreiser** [justin-lau/ember-intl-tel-input#3]
I created a Brocfile.js with the recommended contents. My ember-cli app builds and runs without errors, but utils.js is not loaded (nor is it present in /dist). Is there another requirement I am missing?

Server output:
version: 1.13.1
Could not find watchman, falling back to NodeWatcher for file system events.
Visit http://www.ember-cli.com/#watchman for more info.
Livereload server on http://10.0.1.245:35729
Serving on http://10.0.1.245:4200/

Build successful - 4279ms.

Slowest Trees                                 | Total  
----------------------------------------------+---------------------
Concat: Vendor                                | 3292ms              

Slowest Trees (cumulative)                    | Total (avg)  
----------------------------------------------+---------------------
Concat: Vendor (1)                            | 3292ms  
Babel (3)                                     | 277ms (92 ms)       

Content Security Policy violation: {"csp-report":{"document-uri":"http://10.0.1.245:4200/","referrer":"http://10.0.1.245:4200/","violated-directive":"script-src 'self' 'unsafe-eval' localhost:35729 0.0.0.0:35729","effective-directive":"script-src","original-policy":"default-src 'none'; script-src 'self' 'unsafe-eval' localhost:35729 0.0.0.0:35729; font-src 'self'; connect-src 'self' ws://localhost:35729 ws://0.0.0.0:35729 http://10.0.1.245:4200/csp-report; img-src 'self'; style-src 'self'; media-src 'self'; report-uri http://10.0.1.245:4200/csp-report;","blocked-uri":"http://10.0.1.245:35729","source-file":"http://10.0.1.245:4200/ember-cli-live-reload.js","line-number":6,"column-number":43,"status-code":200}}
Content Security Policy violation: {"csp-report":{"document-uri":"http://10.0.1.245:4200/","referrer":"http://10.0.1.245:4200/","violated-directive":"connect-src 'self' ws://localhost:35729 ws://0.0.0.0:35729 http://10.0.1.245:4200/csp-report","effective-directive":"connect-src","original-policy":"default-src 'none'; script-src 'self' 'unsafe-eval' localhost:35729 0.0.0.0:35729; font-src 'self'; connect-src 'self' ws://localhost:35729 ws://0.0.0.0:35729 http://10.0.1.245:4200/csp-report; img-src 'self'; style-src 'self'; media-src 'self'; report-uri http://10.0.1.245:4200/csp-report;","blocked-uri":"ws://10.0.1.245:35729","source-file":"http://10.0.1.245:35729","line-number":74,"column-number":21,"status-code":200}}

**Comment by justin-lau** [comment]
If you have included `utils.js` in `Brocfile.js`, it will be included in
`dist/vendor.js` when being built. The best way to tell if it is actually
included is to inspect an `{{ember-intl-tel-input}}` component and see if a
placeholder exists, or whether auto formatting is working. Without the
script, neither of them works and only a blank input is displayed.

**Comment by tgreiser** [comment]
'utils.js' is not included in dist/vendor.js and blank input is displayed (I see the flag, but no formatting characters).

My Brocfile.js contains nothing but the snippet from the docs:
var app = new EmberAddon({

  intlTelInput: {
    includeUtilsScript: true, // default to false
  },

});

**Comment by justin-lau** [comment]
Interesting, never found this happened. Can you confirm you have `module.exports = app.toTree();` at the bottom of the file? Also could you please post the versions of your ember and ember-cli?

**Comment by tgreiser** [comment]
As of ember-cli 1.13.0:

Brocfile.js has been deprecated in favor of ember-cli-build.js. See: [TRANSITION.md](https://github.com/ember-cli/ember-cli/blob/master/TRANSITION.md)

Here is my working ember-cli-build.js

``` javascript
var EmberApp = require('ember-cli/lib/broccoli/ember-app');

module.exports = function(defaults) {
  var app = new EmberApp(defaults, {
    // Add options here
    intlTelInput: {
      includeUtilsScript: true,
    },
  });

  return app.toTree();
};
```

**Comment by justin-lau** [comment]
Thanks for the update. Apparently there is a [bug](https://github.com/ember-cli/ember-cli/issues/4435) in `ember-cli` v1.13.1 that prevents options to get merged. This addon won't update its `ember-cli` version until this problem is fixed, in v1.13.2.

**Comment by justin-lau** [comment]
I think this issue has been solved with the v1.2.0 update I published a moment ago. I'm closing this issue for now, but feel free to bring this up again if the problem persists.
